### ISSUE TITLE
sound echo problem

### ISSUE THREAD

**Issue by Anandsaini** [muaz-khan/RTCMultiConnection#1030]
when connecting Bluetooth device or earphones, it is working fine but when connecting with default system speakers, application is echoing a lot. A loud screaming sound is there continuously. Tried so many ways but still problem is there. I am about to give up. Please help in this case. Humble request, please.

**Comment by E1111rong2222** [comment]
I'm happend this problem too， and now has you solved this problem ?

**Comment by Anandsaini** [comment]
Yes. This problem is solved. Keep a sufficient distance between the broadcaster and receiver device. For example, if device1 is using the mic for broadcasting, then device2, device3, etc... which are receiving the output from device1 mic, should be at a sufficient distance(more than 10 Meters)

**Comment by marwinlewis** [comment]
Guys use your brain, when you are on speakers it receive same audio as input through mic hence the echo :D

**Comment by MediaKitApp** [comment]
In addition to all the recommendations, I could tell you that when a local audio is added to the conference, make sure that it is always muted, otherwise you will hear the same audio coming from the microphone, it also happens when you mute/unmute the microphone.
